src/components/navigation-utils.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import routes from '../utils/routes.js';
+import EditIcon from './icons/general/edit-icon.js';
+import UsersIcon from './icons/main-menu/users-icon.js';
+import SettingsIcon from './icons/main-menu/settings-icon.js';
+import DashboardIcon from './icons/main-menu/dashboard-icon.js';
+import { ControlOutlined, QuestionOutlined } from '@ant-design/icons';
+import permissions, { hasUserPermission } from '../domain/permissions.js';
+
+export const getCommonNavigationMenuItems = ({ t, user, helpPage }) => {
```

Comment:
Thank God we finally do not have to repeat this in all apps anymore :heart:

Reply:
Yeees! 🕯️ ✝️